Examples/Messenger/Messenger-Shared/MessageViewController.m

```diff
@@ -26,10 +26,21 @@ @interface MessageViewController ()
 
 @property (nonatomic, strong) NSArray *searchResult;
 
+@property (nonatomic, getter=isStub) BOOL stub;
+
 @end
 
 @implementation MessageViewController
 
++ (instancetype)stub
```

Comment:
It will probably feel better if you have a category that exists in only your testing bundle that creates this stubbed version of the messageController 

```
+ (instancetype)stubbedMessageViewController
{

}
```